### PR TITLE
Update AUTHORS.md and put fabxc into maintainers.

### PR DESCRIPTION
**Pull request by juliusv**

### CODE REVIEW

**Comment by juliusv** [comment]
@beorn7 @fabxc

File: AUTHORS.md [hunk 1]
```diff
@@ -4,22 +4,42 @@ Julius Volz in 2012.
...
+* Deomid Ryabkov <rojer@rojer.me>
+* Fabian Reinartz <fabian.reinartz@soundcloud.com>
 * Johannes 'fish' Ziemke <fish@freigeist.org>
+* Joonas Bergius <joonas@digitalocean.com>
+* Joseph Wilk <joe@josephwilk.net>
 * Julius Volz <julius@soundcloud.com>
+* Julius Volz <julius@soundcloud.com>
+* Marko Mikulicic <mkm@cesanta.com>
 * Matt T. Proud <matt.proud@gmail.com>
-* Peter Bourgon <pb@soundcloud.com>
```
Comment (juliusv) [inline AUTHORS.md]: (Peter is now listed with his non-SoundCloud address)

File: AUTHORS.md [hunk 2]
```diff
@@ -4,22 +4,42 @@ Julius Volz in 2012.
...
+* Miquel Sabaté <mikisabate@gmail.com>
+* Mitsuhiro Tanda <mitsuhiro.tanda@gmail.com>
+* Peter Bourgon <peter@bourgon.org>
 * Sabra Melamed <yosabra@gmail.com>
+* Sam McLeod <sammcj@users.noreply.github.com>
```
Comment (juliusv) [inline AUTHORS.md]: I googled the real name on this one

File: AUTHORS.md [hunk 3]
```diff
@@ -4,22 +4,42 @@ Julius Volz in 2012.
...
 
+* Alan Braithwaite <asbraithwaite@gmail.com>
+* Alexander Staubo <alex@bengler.no>
 * Andres Suarez <andres@soundcloud.com>
 * Bernerd Schaefer <bj.schaefer@gmail.com>
 * Björn Rabenstein <beorn@soundcloud.com>
 * Brian Brazil <brian.brazil@boxever.com>
+* Ceesjan Luiten <ceesjan@ytec.nl>
 * Conor Hennessy <conor@soundcloud.com>
+* Deomid Ryabkov <rojer@rojer.me>
```
Comment (juliusv) [inline AUTHORS.md]: got the real name from a mailing list thread on this one

File: AUTHORS.md [hunk 4]
```diff
@@ -4,22 +4,42 @@ Julius Volz in 2012.
...
+* Fabian Reinartz <fabian.reinartz@soundcloud.com>
 * Johannes 'fish' Ziemke <fish@freigeist.org>
+* Joonas Bergius <joonas@digitalocean.com>
+* Joseph Wilk <joe@josephwilk.net>
 * Julius Volz <julius@soundcloud.com>
+* Julius Volz <julius@soundcloud.com>
```
Comment (beorn7) [inline AUTHORS.md]: Duplicate line.
(Not that two Julius's wouldn't be great for the project...)

Comment (juliusv) [inline AUTHORS.md]: ZOMG. Fixed. I'll update this again once I've cloned myself.

**Comment by beorn7** [comment]
:+1:

File: AUTHORS.md (hunk 4 above)
Comment (fabxc) [inline AUTHORS.md]: The so called "Julei".

**Pull request merged**